Author: Piotr Galiszewski <piotrgaliszewski@gmail.com>
Description: [PATCH] Revert "protocols.
 Only send jabber status to server once when connecting
---
diff --git a/kadu-core/protocols/protocol-state-machine.cpp b/kadu-core/protocols/protocol-state-machine.cpp
index dc183c9..e34cd17 100644
--- a/kadu-core/protocols/protocol-state-machine.cpp
+++ b/kadu-core/protocols/protocol-state-machine.cpp
@@ -38,7 +38,7 @@
 ProtocolStateMachine::ProtocolStateMachine(Protocol *protocol) :
 		QStateMachine(protocol), CurrentProtocol(protocol)
 {
-	DelayTimer.setInterval(500);
+	DelayTimer.setInterval(1000);
 	DelayTimer.setSingleShot(true);
 
 	LoggingOutState = new QState(this);
diff --git a/kadu-core/protocols/protocol.cpp b/kadu-core/protocols/protocol.cpp
index c0c50a4..3ed8828 100644
--- a/kadu-core/protocols/protocol.cpp
+++ b/kadu-core/protocols/protocol.cpp
@@ -222,6 +222,7 @@ void Protocol::loggedInStateEntered()
 	afterLoggedIn();
 
 	statusChanged(loginStatus());
+	sendStatusToServer();
 
 	emit connected(CurrentAccount);
 }
diff --git a/plugins/gadu_protocol/gadu-protocol.cpp b/plugins/gadu_protocol/gadu-protocol.cpp
index 1375646..53a0823 100644
--- a/plugins/gadu_protocol/gadu-protocol.cpp
+++ b/plugins/gadu_protocol/gadu-protocol.cpp
@@ -252,7 +252,6 @@ void GaduProtocol::afterLoggedIn()
 	setUpFileTransferService();
 
 	sendUserList();
-	sendStatusToServer();
 }
 
 void GaduProtocol::logout()
